CMakeLists.txt

```diff
@@ -280,7 +280,15 @@ if(NOT DEFINED SNMALLOC_ONLY_HEADER_LIBRARY)
   foreach(TEST_CATEGORY ${TEST_CATEGORIES})
     subdirlist(TESTS ${TESTDIR}/${TEST_CATEGORY})
     foreach(TEST ${TESTS})
-      foreach(SUPER_SLAB_SIZE 1;16;oe)
+      if (WIN32 OR APPLE)
```

Comment:
I still think FreeBSD should be included here it fails just as Darwin despite posix_memalign.

Reply:
The failure on FreeBSD isn't a FreeBSD bug, it's a bug in the realloc implementation.  We allocate 4KiB with 4KiB alignment but then `malloc_useable_size` returns 16KiB, so we don't reallocate and copy.  We then end up with a 4KiB-aligned 16KiB allocation.  We should be reallocating if the result of `malloc_useable_size` is more than three times the alignment of the allocation.  I don't know if the macOS failures have the same cause.

Reply:
Otherwise OpenBSD does not have malloc*size api nor any similar-ish call to it.

Reply:
Fixed Apple, so hopefully FreeBSD too.  Should I disable OpenBSD for these tests as the tests won't compile?

Reply:
It might be better to disable for OpenBSD indeed.

Reply:
I believe I have disabled OpenBSD. I also disabled NetBSD as I could not find a malloc*size API in the documentation.

Reply:
There is sort of malloc_usable_size present in the libc symbols but not exposed most likely not functional yet,so you re right.